### DISCUSSION TITLE
Not a single app is float

### DISCUSSION THREAD

**Discussion by moresol** [asmvik/yabai#2078] [Q&A] [answered]
MacOS Sonoma latest. 
In verbose I see these errors, how do I fix it?
SIP is turned off.
Yabai doesn't work correctly.
I tried to reinstall, to reboot, no luck.
Mouse follows focus works incorrectly, float windows are on top of other windows. Please, help.
I've seen this post https://github.com/koekeishiya/yabai/issues/575
It says there's nothing wrong with the errors. So, what can it be?

![Screenshot 2024-02-04 at 2 37 19 AM](https://github.com/koekeishiya/yabai/assets/106124331/f6ca3c28-00c5-4ce1-a309-b8fbd9a737df)
![Screenshot 2024-02-04 at 2 27 41 AM](https://github.com/koekeishiya/yabai/assets/106124331/a83119d6-78b4-4c55-a001-972dfd6abc23)

**Comment by asmvik** [comment, marked as answer]
Add `yabai -m rule --add app=".*" layer=normal` if you want floating windows to NOT stay above other windows.

**Reply by moresol** [reply]
I tried this yabai -m rule --add app="Telegram" layer=normal
Doesn't work. Am I missing something?

**Reply by moresol** [reply]
I didn't notice before, it's even worst.
None of those apps are float at all, even though, the rules are set, and scripting addition is updated.
I remember, a few weeks it was fine.
How do I fix it? 😖
![Screenshot 2024-02-07 at 3 41 29 AM](https://github.com/koekeishiya/yabai/assets/106124331/b4221563-6b8f-4165-9e00-b9bbe9315abb)
